tests/sles4sap/trento/test_hana_unregister_node.pm

```diff
@@ -0,0 +1,82 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Trento unregister the secondary node
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use base 'consoletest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils 'script_retry';
+use qesapdeployment;
+use trento;
+
+sub run {
+    my ($self) = @_;
+    select_serial_terminal;
+
+    my $secondary_host = 'vmhana02';
+    cluster_print_cluster_status($secondary_host);
+
+    # Set secondary node in maintenance mode
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    qesap_ansible_cmd(cmd => "sudo crm configure property maintenance-mode=true",
+        provider => $prov,
+        filter => $secondary_host);    
+
+    # Stop hana instance
+    $cmd_stop = join(' ', 'HDB', 'stop');
+    cluster_hdbadm($secondary_host, $cmd_stop);
+
+    # Unregistered the hana database and started
+    my $cmd_unregister = join(' ', 'hdbnsutil',
+        '-sr_unregister',
+        '--name=miky');
+    my $cmd_start = join(' ', 'HDB', 'start');        
+    cluster_hdbadm($secondary_host, $cmd_unregister);
+    cluster_hdbadm($secondary_host, $cmd_start);
```

Comment:
Like before, directly place the command string in this one too

Reply:
Also here, do we not need to wait for Trento to keep up?

Reply:
wait applied